### PR TITLE
Adding monitor-server-api in conda-forge

### PR DESCRIPTION
**Pull request by js-dieu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by js-dieu** [comment]
I hereby confirm I am the maintainer of this recipe

File: recipes/monitor-server-api/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  sha256: {{ sha256 }}
+  md5: {{ md5 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build: []
```
Comment (mwcraig) [inline recipes/monitor-server-api/meta.yaml:22]: You can drop the `build` section

File: recipes/monitor-server-api/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  requires:
+    - pip
+  commands:
+    - pip check
+    - type launch-monitor-server.sh  # [not win]
+
+about:
+  home: https://github.com/CFMTech/{{ name }}
+  license: MIT
+  license_file: LICENSE.txt
```
Comment (mwcraig) [inline recipes/monitor-server-api/meta.yaml:50]: If you change this to `LICENSE` then conda will find the license inside your source package.

**Review by mwcraig — COMMENTED** [review]
Thanks for opening this, @js-dieu! You can remove the license file from this pull request as long as you make the change I suggested in the `about` section of the recipe. The preference is to get the license from the source distribution whenever possible.

(also, the PyPI badge on tour source repo links to an image of the badge not to the pypi page for the package, but that is unrelated to packaging here 😀 )

File: recipes/monitor-server-api/meta.yaml (hunk 1 above)
Comment (js-dieu) [reply recipes/monitor-server-api/meta.yaml]: done

Comment (js-dieu) [reply recipes/monitor-server-api/meta.yaml]: Thanks for spotting the badge issue.  🥇 
Regarding the LICENSE point you raised, I wasn't aware of this "trick". Thanks for point it out 👍🏻

File: recipes/monitor-server-api/meta.yaml (hunk 2 above)
Comment (js-dieu) [reply recipes/monitor-server-api/meta.yaml]: done

**Comment by mwcraig** [comment]
Closing/reopening to trigger fresh CI builds

**Comment by mwcraig** [comment]
Thanks @js-dieu , merging

**Pull request merged**